src/ctelnet.cpp

```diff
@@ -1914,6 +1914,11 @@ void cTelnet::handle_socket_signal_readyRead()
         }
         else
         {
+            if( ch == TN_BELL )
```

Comment:
With the Lua `alert(...)` command I would consider not putting any code here at all and allow those who want it to code a simple (sub-string) trigger (on `\07`) themselves.